dfid-transition/config.ru

```diff
@@ -2,7 +2,7 @@ require 'sidekiq'
 
 Sidekiq.configure_client do |config|
   config.redis = {
-    host: ENV["REDIS_HOST"] || "127.0.0.1",
+    host: "redis-1.backend",
```

Comment:
I think localhost is used for local development. Perhaps it's clearer in the short term if we hard code it like this?